CHANGELOG.md

```diff
@@ -215,7 +215,7 @@
 ### Chore & Maintenance
 
 - `[docs]` Improve documentation of `mockClear`, `mockReset`, and `mockRestore` ([#6227](https://github.com/facebook/jest/pull/6227/files))
-- `[jest-circus]` Add dependency on jest-each ([#6309](https://github.com/facebook/jest/pull/#6309))
+- `[jest-circus]` Add dependency on jest-each ([#6309](https://github.com/facebook/jest/pull/6309))
```

Comment:
Is this line a duplicate of 221? Maybe remove one of the lines and fix the link?

Reply:
You're right. I've removed the duplicate and pushed.

Reply:
btw the _other_ link is broken the same way this one was

Reply:
Are you sure? The remaining one looks okay.

Reply:
you're right, my b